src/content/configuration/other-options.mdx

```diff
@@ -551,6 +551,63 @@ Options for infrastructure level logging.
 
 `object = {}`
 
+### appendOnly
+
+<Badge text='5.31.0+' />
+
+`boolean`
+
+Only appends lines to the output. Avoids updating existing output e. g. for status messages. This option is only used when no custom [`console`](#console) is provided.
```

Comment:
```suggestion
Append lines to the output instead of updating existing output, useful for status messages. This option is used only when no custom [`console`](#console) is provided.
```